src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -149,14 +149,14 @@ public void Node_RenderingUpToDate()
             p1.UpdateValue(new UpdateValueParams("IsVisible", "false"));
 
             Assert.True(BackgroundPreviewGeometry.HasNumberOfPointsCurvesAndMeshes(7, 6, 0));
-            Assert.AreEqual(6, BackgroundPreviewGeometry.NumberOfInvisiblePoints());
+            Assert.AreEqual(1, BackgroundPreviewGeometry.NumberOfInvisiblePoints());
```

Comment:
@ikeough these changes in tests are done to match new behavior which is presented in this PR. All points of a node are stored in a single geomentry instead of storing each array item of the node separately. So, here are 6 points which are stored in 1 `PointGeometryModel3D` from now